software/control/gui_hcs.py

```diff
@@ -336,13 +348,12 @@ def setupHardware(self):
             self.camera_focus.enable_callback()
             self.camera_focus.start_streaming()
 
-    def waitForMicrocontroller(self, timeout=None, error_message=None):
```

Comment:
This change is dual purpose: first, don't re-write code if it already exists.  and second, before this implementation would hang forever if timeout=None.